iroh-gateway/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
Please add a `README` with instructions how to run this and a one sentence description of what this is